### DISCUSSION TITLE
Translate strings?

### DISCUSSION THREAD

**Discussion by man-tax** [Budibase/budibase#5526] [Help] [answered]
And another question. Is it possible to translate the strings appearing every here and there? Is there a file with all the strings somewhere that can be changed? I would volunteer for German.

![image](https://user-images.githubusercontent.com/104101494/164548875-0f13308d-2d35-4bc3-8ed2-6c51e78012aa.png)

Thanks
Sebastian

**Comment by melohagan** [comment, marked as answer]
Hey @man-tax 

I think this feature is what you are describing: https://github.com/Budibase/budibase/issues/3950

**Reply by man-tax** [reply]
Thanks! Such feature would be great. Besides though, I was thinking maybe there is an easy way of just changing the strings for my installation in a file somewhere and someone tells me where to find it ;)

**Comment by albert0346** [comment]
I am also interested in translating the text in the end user interface. I like Budibase very much but when this feature is not available I have to look for another solution.

**Reply by albert0346** [reply]
Or do you know perhaps a platform with the same or even better functionality with German end user interface or with the ability to translate it to German?

**Reply by tglynx** [reply]
Hi Albert, 
i was facing the same issue. However, in most cases you can just disable the 'default notification' and then display a custom notification using a action. like this: 

Do not display default notification
![Bildschirm­foto 2023-07-25 um 08 22 27](https://github.com/Budibase/budibase/assets/30650435/28fb0ba4-b939-4431-b482-bbe04f72552f)

Instead use custom notification
![Bildschirm­foto 2023-07-25 um 08 23 28](https://github.com/Budibase/budibase/assets/30650435/9d644809-9284-4217-97ef-ade3f9d0edb5)

However, this may be lots of effort to catch it in all situations, i know. A central 'String Translation' / 'String Ressources' concept for multilanguage support would be so nice 👍 

Hope this helps!

**Comment by albert0346** [comment]
Thanks Michael!

Yes, as you wrote: "lots of effort to catch it in all situations"!

Am Di., 25. Juli 2023 um 08:26 Uhr schrieb Michael Sommer <
***@***.***>:

**Reply by albert0346** [reply]
Is here a budibase admin who can say me WHEN (as exact as possible), from now on, we can translate the strings into our language.

I need this exact information to make a decision if I should continue to work with budibase or look for another solution where I can translate strings.

Or when this date is TOO far away from now, does anyone know another free low-code platform where I can translate strings?

**Reply by melohagan** [reply]
Hey @albert0346 This is a feature we definitely want to add at some future date, but it's not likely to be within the next few months, and I can't give an exact deadline. At a glance, ToolJet may offer this feature

**Reply by albert0346** [reply]
Thanks melohagan!
